client/utils/gatsby/guidePageCreator.js

```diff
@@ -32,7 +32,14 @@ exports.createGuideArticlePages = createPage => ({
 
   return createPage({
     path: `/guide${slug}`,
-    component: guideArticle,
+    component: !component
+      ? guideArticle
+      : path.resolve(
+          __dirname,
+          '../../src/templates/Guide/components/',
+          component,
+          'index.js'
```

Comment:
I'm not sure appending `index.js` on the supplied `path/to/component`is the best way to go and may cause confusion.  We should probably take it as the component is located where `path/to/component` points to.

Reply:
In that case in an md-file, we should use `path/to/component/EntryPoint.js`. In the example, `EntryPoint` is `index`. Right?

Reply:
Sounds good to me.